unic/ucd/normal/src/gen_cat.rs

```diff
@@ -13,12 +13,10 @@
 mod mark {
     use std::cmp::Ordering;
 
-    const GENERAL_CATEGORY_MARK: &'static [(char, char)] =
-        include!("tables/general_category_mark.rsv");
-
     /// Return whether the given character is a combining mark (`General_Category=Mark`)
     pub fn is_combining_mark(c: char) -> bool {
-        GENERAL_CATEGORY_MARK.find(c).is_some()
+        const TABLE: &'static [(char, char)] = include!("tables/general_category_mark.rsv");
+        TABLE.find(c).is_some()
```

Comment:
I got it working with `use unic_utils::CharDataTable;` under `mod mark` and `CharDataTable::<()>::find(&TABLE, c).is_some()`.

Unfortunately `&[(char, char)]` matches `&[(char, V)]` as well, thus the reason I want to get around to adding the `CharRange` structure.

Once all of `unic-gen` is engaged and I can make the transition to actually using it across the board, I'll work on that.